modules/gathering-clusterversion-history-cli.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history-cli_{context}"]
+= Gathering ClusterVersion history using the {oc-first}
+
+To view the ClusterVersion history using the {oc-first}:
```

Comment:
```suggestion
You can view the ClusterVersion history by using the {oc-first}.
```

Reply:
Should be a sentence because the next item is a section. Also, see [IBM Style guidance](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u) on "by using".